src/main/java/propertyFiles/Bundle.properties

```diff
@@ -2715,6 +2715,10 @@ rtabfileparser.ioexception.read=Couldn't read Boolean variable ({0})!
 rtabfileparser.ioexception.parser1=R Tab File Parser: Could not obtain varQnty from the dataset metadata.
 rtabfileparser.ioexception.parser2=R Tab File Parser: varQnty=0 in the dataset metadata!
 
+#JsonParser.java
+jsonparser.error.metadatablocks.not.found=Invalid json object: metadata blocks not found.
+jsonparser.error.parsing.date=Error parsing date:
+jsonparser.error.parsing.number=Error parsing number:
```

Comment:
FWIW: Security scans have started disliking when the see raw user text reflected in the response, so I've started dropping the idea of sending their bad/possibly malicious text back. (I think the scans are assuming API calls used to create the UI where bad text could be an issue, versus command/line script use. That said, possibly easier to avoid having to explain.)

Reply:
Good point.

Reply:
I made the suggested change before I saw these comments should I revert it?

Reply:
Actually, this isn't raw user text, is it? It's the message being returned from the exception. (Which may include the raw user text? I don't know.)

I don't feel strongly about it. @qqmyers do you want to make a call on this?

Reply:
I didn't want to make a call but - yeah, as long as this is just the exception message, my comment doesn't really apply. I just assumed the "Error parsing date:" was going to show the date. Showing the exception mesage should be safe and more helpful than not explaining.  (hopefully low-level exceptions don't put raw text in their messages)

Reply:
Ok, let's stick with showing the exception message. Hopefully it's informative to the user.